docs/sources/enterprise/reporting.md

```diff
@@ -106,48 +106,26 @@ When the CSV file is generated, it is temporarily written to the `csv` folder in
 A background job runs every 10 minutes and removes temporary CSV files. You can configure how long a CSV file should be stored before being removed by configuring the [temp-data-lifetime]({{< relref "../administration/configuration/#temp-data-lifetime" >}}) setting. This setting also affects how long a renderer PNG file should be stored.
 
 ### Scheduling
+> Note: Scheduler has been significantly changed in Grafana Enterprise v8.1.
 
-Scheduled reports can be sent on a monthly, weekly, daily, or hourly basis. You may also disable scheduling for when you either want to send it via the API.
+Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals.  You can also disable scheduling by selecting **Never** if you want to send the report via the API.
```

Comment:
```suggestion
Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals.  You can also disable scheduling by selecting **Never** or if you want to send the report via the API.
```

Reply:
Not sure about this. I meant to say that "Never" can be used to disable scheduling, for example, when report distribution needs to be triggered via API.

Reply:
What I meant is that in the current state it reads as if you'd only set schedule to Never to send reports via API, but sometimes you'd do that just not to send a report. I might have misunderstood the sentence though.

Reply:
You're right. Thank you!